.circleci/config.yml

```diff
@@ -24,16 +23,15 @@ jobs:
         keys:
         - cached-dependencies
         paths:
-        - frontend/node_modules
```

Comment:
Reconsider this.

Reply:
I'm not sure about it too. We can do a test without removing it.